src/edu/psu/sweng500/team8/gui/CustomKeyListener.java

```diff
@@ -24,7 +24,10 @@ public CustomKeyListener(Cell cell, GameSession gameSession) {
 
 	public void keyTyped(java.awt.event.KeyEvent keyEvent) {
 		
-		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar()) {
+		
+		
+		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar() || KeyEvent.VK_DELETE == keyEvent.getKeyChar()) {
+			
 			gameSession.enterNumber(this.cell, 0);
 		} else {
 			gameSession.enterNumber(this.cell, Integer.parseInt(String.valueOf(keyEvent.getKeyChar())));

```

Comment:
this seems a bit risky if this code is hit before David's enforceValidNumbers action listener. I think we should unify the logic of validating the entered key and setting the number on the cell.